src/libcrun/seccomp.h

```diff
@@ -26,7 +26,12 @@
 # include <oci_runtime_spec.h>
 # include "container.h"
 
-int libcrun_generate_and_load_seccomp (libcrun_container_t *container, int outfd, char **flags, size_t flags_len, libcrun_error_t *err);
+enum
+  {
+    LIBCRUN_SECCOMP_FAIL_UNKNOWN_SYSCALL = 1 << 0,
```

Comment:
why shift by 0? or am I misreading here

Reply:
no misreading, it is really a shift by 0 :-)  It is a nop but it makes clear it is a bitmask